src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -921,6 +930,24 @@ private MenuBar createMenu() {
         return menu;
     }
 
+    private ProgressIndicator createTaskIndicator() {
+        ProgressIndicator indicator = new ProgressIndicator(1);
+        indicator.progressProperty().bind(stateManager.tasksProgressBinding);
+
+        indicator.setOnMouseClicked(new EventHandler<MouseEvent>() {
+            @Override
+            public void handle(MouseEvent event) {
+                TaskProgressDialog taskProgressDialog = new TaskProgressDialog();
```

Comment:
Instead of using a real dialog, what about using a collapse overlay similar to how it's done in firefox?
https://github.com/controlsfx/controlsfx/wiki/ControlsFX-Features#popover

Reply:
Uh fancy! I'll look into it!

Reply:
![Screenshot from 2020-05-08 15-12-50](https://user-images.githubusercontent.com/10479048/81409085-c1952b80-913e-11ea-9935-36c17d30931d.png)
That looks much better! Great idea!